Framework/DataHandling/test/SaveNexusGeometryTest.h

```diff
@@ -0,0 +1,193 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+//     NScD Oak Ridge National Laboratory, European Spallation Source
+//     & Institut Laue - Langevin
+// SPDX - License - Identifier: GPL - 3.0 +
+#ifndef MANTID_DATAHANDLING_SAVENEXUSGEOMETRYTEST_H_
+#define MANTID_DATAHANDLING_SAVENEXUSGEOMETRYTEST_H_
+
+#include "MantidAPI/IEventWorkspace.h"
+#include "MantidAPI/ITableWorkspace.h"
+#include "MantidAPI/WorkspaceFactory.h"
+#include "MantidAPI/WorkspaceProperty.h"
+#include "MantidDataHandling/LoadEmptyInstrument.h"
+#include "MantidDataHandling/SaveNexusGeometry.h"
+#include "MantidKernel/ProgressBase.h"
+#include "MantidTestHelpers/FileResource.h"
+#include "MantidTestHelpers/WorkspaceCreationHelper.h"
+
+#include <H5Cpp.h>
+#include <boost/filesystem.hpp>
+#include <cxxtest/TestSuite.h>
+
+using Mantid::DataHandling::SaveNexusGeometry;
+
+class SaveNexusGeometryTest : public CxxTest::TestSuite {
+public:
+  // This pair of boilerplate methods prevent the suite being created statically
+  // This means the constructor isn't called when running other tests
+  static SaveNexusGeometryTest *createSuite() {
+    return new SaveNexusGeometryTest();
+  }
+  static void destroySuite(SaveNexusGeometryTest *suite) { delete suite; }
+
+  void test_Init() {
+    SaveNexusGeometry alg;
+    TS_ASSERT_THROWS_NOTHING(alg.initialize())
+    TS_ASSERT(alg.isInitialized())
+  }
+
+  void test_exec() {
+
```

Comment:
There are a lot of these excess newlines at the start of functions that hurt readability.